documentation/user-interface/fixture-view.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: Fixture View
+permalink: /documentation/user-interface/fixture-view.html
+---
+# Fixture View
+
+The **Fixture** view displays information about a fixture: the fixture name, tested webpage's URL and list of included tests.
+
+![Fixture view](../../images/user-interface/fixture-view.png)
```

Comment:
May be better use screenshot with hovered test item to show `delete` and `run` button for particular test